src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -368,6 +368,36 @@ public function preprocessEventDetail(Event $event, string $langcode, array $set
         return $variables;
     }
 
+    /**
+     * Get the default image
+     *
+     * @param array $settings
+     * @param string $eventTypeId
+     * @return string $defaultImage
+     */
+    public function getDefaultImage($settings, $eventTypeId)
+    {
+        if (!$settings['enabled']) {
+            return '';
+        }
+
+        if ($settings['type'] === 'uit') {
+            return $this->request->getScheme() . '://media.uitdatabank.be/static/uit-placeholder.png';
+        }
+
+        if ($settings['type'] === 'theme') {
+            $fallbackImages = Yaml::parse(file_get_contents(__DIR__ . '/../../../fallback_images.yml'));
```

Comment:
@konrad126 / @JonasVHG 
Not sure about this line.
Would it have an impact on the performance to read this file for every event?

Reply:
You could always store it in a local property and parse the yml if the local property is empty